cdap-app-fabric/src/main/java/io/cdap/cdap/scheduler/ScheduleNotificationSubscriberService.java

```diff
@@ -275,4 +291,12 @@ protected void processNotification(ProgramScheduleStoreDataset scheduleStore,
       }
     }
   }
+
+  private boolean isApplicationDisabled(String namespace, String application) throws IOException {
```

Comment:
Change the method to perform positivity check.

Reply:
Done